### ISSUE TITLE
Tagging plugin crashes Javascript in IE11

### ISSUE THREAD

**Issue by Chrisgemini** [cosmocode/tagging#72]
I run Dokuwiki Hogfather on a Ubuntu server with some 30+ major plugins installed.
I access the Wiki thru Windows 10 and the built-in IE11. 
With Tagging plugin deactivated, the wiki runs the Javascript of the template and all the other plugins on Firefox and IE11 without problems.
When I activate Tagging plugin, Javascript completely freezes in IE11. In the console I can't find any Javascript errors, though. 
In Firefox, however, everything works fine, again.
So, in summary, the recent version of the Tagging plugin seems to be incompatible with IE11, javascript-wise.
